tests/behat/features/prof/03-report/06-deputy-code-estimates.feature

```diff
@@ -83,13 +83,17 @@ Feature: Prof deputy costs estimate
     When I click on "breadcrumbs-report-overview"
     Then I should see the "prof_deputy_costs_estimate-state-done" region
 
+    @tdd
   Scenario: Completing the Assessed Costs route with costs and more info and viewing the status overview
     Given I am logged in as "behat-prof1@publicguardian.gov.uk" with password "Abcd1234"
     And I click on "pa-report-open" in the "client-01000014" region
     And I click on "edit-prof_deputy_costs_estimate"
     When I click on "start"
     And the step with the following values CAN be submitted:
       | deputy_costs_estimate_profDeputyCostsEstimateHowCharged_0 | assessed |
+    And the URL should match "/report/\d+/prof-deputy-costs-estimate/management-costs"
```

Comment:
add a validation check here, something like "and the following values cannot be submitted" then try sending blank value. This should pass as validation should not allow empty data to be submitted. See other tests if unsure, or ask.